src/pageTypes/components/PageTypeDetailsPage/PageTypeDetailsPage.tsx

```diff
@@ -60,7 +60,7 @@ const useStyles = makeStyles(
 const PageTypeDetailsPage: React.FC<PageTypeDetailsPageProps> = props => {
   const {
     disabled,
-    errors,
+    errors = [],
```

Comment:
I don't have strong opinion on this, but I think we should use `Component.defaultProps` for this, so it can be viewed in react inspector :)